src/test/librbd/test_notify.py

```diff
@@ -140,21 +140,23 @@ def slave(ioctx):
         assert(not image.is_exclusive_lock_owner())
         assert(list(image.list_snaps()) == [])
 
-        print("update_features")
-        assert((image.features() & RBD_FEATURE_OBJECT_MAP) != 0)
-        image.update_features(RBD_FEATURE_OBJECT_MAP, False)
-        assert(not image.is_exclusive_lock_owner())
-        assert((image.features() & RBD_FEATURE_OBJECT_MAP) == 0)
-        image.update_features(RBD_FEATURE_OBJECT_MAP, True)
-        assert(not image.is_exclusive_lock_owner())
-        assert((image.features() & RBD_FEATURE_OBJECT_MAP) != 0)
-        assert((image.flags() & RBD_FLAG_OBJECT_MAP_INVALID) != 0)
-
         print("rebuild object map")
         image.rebuild_object_map()
         assert(not image.is_exclusive_lock_owner())
         assert((image.flags() & RBD_FLAG_OBJECT_MAP_INVALID) == 0)
 
+        disable_update_features = int(os.getenv("DISABLE_UPDATE_FEATURES"))
+        if disable_update_features is None or not disable_update_features):
```

Comment:
I don't think it necessary should be an int. Just any non-empty value. So you could just check `if not disable_update_features`. And I expect `int(empty_string)` will throw an exception. Probably you can use `bool(os.getenv("DISABLE_UPDATE_FEATURES"))`.

And I suggest to add "RBD_" prefix to the environmental variable; "RBD_DISABLE_UPDATE_FEATURES".

Reply:
or, just put
```py
if 'DISABLE_UPDATE_FEATURES' in os.environ:
  # ...
```

Reply:
makes things simpler! updated thanks!